parquet-hadoop/src/test/java/org/apache/parquet/format/converter/TestParquetMetadataConverter.java

```diff
@@ -252,4 +267,47 @@ public void randomTestFilterMetaData() {
     }
   }
 
+  @Test
+  public void testNullFieldMetadataDebugLogging() throws NoSuchFieldException, IllegalAccessException, IOException {
```

Comment:
thanks for pointing that out, I didn't see that your PR already fixes the same issue. For now I will update the unit test below and once your PR is in I can rebase this PR.